website/members/templates/members/announcement_not_member.html

```diff
@@ -1,4 +1,8 @@
 <p>
     <i class="fas fa-id-card me-2"></i>
-    You're currently not a member of Thalia. <a href="{% url 'registrations:renew' %}">Renew your membership</a> to get access to all parts of the website.
+    {% if member.latest_membership.study_long and not member.profile.is_minimized %}
+        You're currently not a member of Thalia. <a href="{% url 'registrations:renew-studylong' %}">Renew your membership</a> to get access to all parts of the website.
```

Comment:
Maybe the text could reflect that it's not really renewing, just telling us you're still a student.